cmd/soci/commands/ztoc/info.go

```diff
@@ -28,6 +29,18 @@ import (
 	"oras.land/oras-go/v2/content/oci"
 )
 
+type ZtocInfo struct {
```

Comment:
Maybe add JSON tags directly to the structs in https://github.com/awslabs/soci-snapshotter/blob/main/ztoc/ztoc.go?

Reply:
```suggestion
type Info struct {
```

to fix the lint error

Reply:
This will probably require some hacks with the reflection package though.